elements/rh-breadcrumb/rh-breadcrumb.ts

```diff
@@ -49,16 +60,80 @@ export class RhBreadcrumb extends LitElement {
    */
   @property({ reflect: true }) variant?: 'subtle';
 
-  static readonly styles = [styles];
+  /**
+ * Breadcrumbs over four items will be truncated and include a button to expand the middle breadcrumb items
+ */
+  @property({ reflect: true, type: Boolean }) truncate? = false;
+
+  #list: HTMLOListElement | null = null;
+  #listItems: NodeListOf<HTMLLIElement> | null | undefined = null;
 
   render() {
     const label = this.accessibleLabel ? this.accessibleLabel : 'Breadcrumb';
     return html`
-      <nav aria-label="${label}" part="container" id="container">
+      <nav 
+        aria-label="${label}"
+        @click="${this.truncate ? this.#handleTruncationClick : undefined}"
+        @keyup="${this.truncate ? this.#onKeyUp : undefined}"
+        id="container"
+        part="container">
         <slot></slot>
       </nav>
     `;
   }
+
+  firstUpdated(): void {
+    this.#updateLightDOM();
+  }
+
+  #updateLightDOM(): void {
+    if (isServer || !this.truncate) {
+      return;
+    }
+
+    this.#list = this.querySelector('ol');
+    if (!this.#list) {
+      return;
+    }
+
+    if (this.#list.children.length < 5) {
+      return;
+    }
+
+    const middleItems = this.#list.querySelectorAll('li:nth-child(n+2):nth-last-child(n+3)');
+    for (const item of middleItems) {
+      item.setAttribute('hidden', 'true');
+    }
+
+    const [truncationBtnString] = truncationBtn.strings;
```

Comment:
Accessing the strings property here doesn't seem proper.

It might be better here to render() to a throwaway document fragment, or use a template element.

Even better would be to render this button in the shadow root